modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -250,7 +251,7 @@ export default class Tile3DLayer extends CompositeLayer {
         data: SINGLE_DATA,
         getPosition: [0, 0, 0],
         getColor: getSimpleMeshLayerColor(tileHeader),
```

Comment:
What is this new prop for and why is it not documented?

Reply:
https://github.com/visgl/deck.gl/pull/5814